docs/data/tree-view/rich-tree-view/selection/selection.md

```diff
@@ -17,6 +17,16 @@ The Tree View also supports multi-selection:
 
 {{"demo": "MultiSelectTreeView.js"}}
 
+## Checkbox selection
+
+To activate checkbox selection set `checkboxSelection={true}`:
```

Comment:
```suggestion
To display the selection using a checkbox set `checkboxSelection={true}`:
```

Btw, I think that the default style for selected `treeItems` should be **either** using a checkbox or using colored rows background. I'd argue that using both feedback feels redundant and may not be what users expect in most cases (ref: [original issue](https://github.com/mui/mui-x/issues/6019)).

What do you think?

Reply:
I used the wording of the data grid doc, do you want me to apply your suggestion there as well?

> Btw, I think that the default style for selected treeItems should be either using a checkbox or using colored rows background. I'd argue that using both feedback feels redundant and may not be what users expect in most cases (ref: https://github.com/mui/mui-x/issues/6019).

You are entirely right, I'll fix it :+1:

Reply:
Should be fixed

Reply:
Nodes without children should probably not have any hover effect and the cursor should be different since they are not interactive

Reply:
> I used the wording of the data grid doc, do you want me to apply your suggestion there as well?

It's a small thing, so I'm not sure it's worth it alone, but I think "activates" doesn't describe very precisely the behavior/outcome from setting this property.

Reply:
I do agree that your wording sounds better